src/utils/amortization.utils.ts

```diff
@@ -0,0 +1,78 @@
+import { DAYS_IN_REPAYMENT_PERIOD } from "@consts";
+import { Ratio } from "@credix/credix-client";
+import Big from "big.js";
+import { round } from "./format.utils";
+
+export const calculateMonthlyPayment = (
+	principal: number,
+	rate: Ratio,
+	numberOfPayments: number
+) => {
+	if (!rate) {
+		rate = new Ratio(1, 1);
+	}
+
+	if (!numberOfPayments) {
+		numberOfPayments = 1;
+	}
```

Comment:
Can these two things ever happen? Shouldn't we always pass valid values to this method?